big_tests/tests/metrics_roster_SUITE.erl

```diff
@@ -51,9 +51,7 @@ subscription_tests() -> [subscribe,
 
 init_per_suite(Config) ->
     instrument_helper:start(declared_events()),
-    MongooseMetrics = [{[global, data, xmpp, received, xml_stanza_size], changed},
```

Comment:
Some tests are effectively removed here, but I guess they were redundant in roster tests.

Reply:
correct, it felt xml_stanza_size was just randomly added here before.